odoo/addons/base/models/ir_ui_view.py

```diff
@@ -2687,7 +2687,7 @@ def _get_view_field_attributes(self):
         return [
             'context', 'currency_field', 'definition_record', 'digits', 'domain', 'group_operator', 'groups', 'help',
             'name', 'readonly', 'related', 'relation', 'relation_field', 'required', 'searchable', 'selection', 'size',
-            'sortable', 'store', 'string', 'translate', 'trim', 'type',
```

Comment:
I would rather like that you keep the list in the alphabetical order, even if it means you will have to touch multiple lines instead of one.
And keep it less than 120 chars